### ISSUE TITLE
L1 Regression Task

### ISSUE THREAD

**Issue by julian-carpenter** [Sara-Ahmed/SiT#11]
Hi Sara,

shouldn't you use 
`torch.exp(-.5*_s) * loss + 0.5 * _s`
in line 51 for the L1 loss, as you want to minimize 1/alpha_1 and not 1/(2*alpha_1**2), as it is right now?

https://github.com/Sara-Ahmed/SiT/blob/ffc7317f046bc21498356355c4e947504677f630/losses.py#L32-L51

Thanks in advance for your answer!

EDIT 23.07:
I edited my question correcting an error in my proposal.

**Comment by Sara-Ahmed** [comment]
Yes you are right, adj should be inside not outside. Actually in our experiments we even experiments with adj=1 in all cases, and this did not affect the accuracies. But out of correctness, we used the 0.5. Thanks for pointing it out, I will fix it now.
